src/components/notes/Editor.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ReactMde from "react-mde";
+import Showdown from "showdown";
+import "react-mde/lib/styles/css/react-mde-all.css";
+
+export default function Editor({ currentNote, updateNote }) {
+  const [selectedTab, setSelectedTab] = React.useState("write");
```

Comment:
useState is exported from react use the one that is exported
```suggestion
  const [selectedTab, setSelectedTab] = useState("write");
```